src/app.js

```diff
@@ -1 +1,10 @@
+/* eslint-disable no-console */
 'use strict';
+
+const fs = require('fs');
+
+fs.cp('./src/1.txt', './src/1 copy.txt', (err) => {
```

Comment:
The code is missing a check to ensure that source and destination files are not the same. This could potentially lead to unwanted behavior. Consider adding a check to avoid this scenario.